src/components/iconinitials/iconinitials.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import "./iconinitials.scss";
+
+export type IconInitialsProps = React.HTMLAttributes<HTMLDivElement> & {
+  initials: string;
+  size?: string;
+  fontSize?: string;
+};
+
+/**
+ * IconInitials component. Used to display an icon to show initial letters.
+ * @param initials
+ * @param props
+ * @constructor
+ */
+export const Iconinitials: React.FC<IconInitialsProps> = ({
+  initials,
+  size = "2em",
+  fontSize,
+  ...props
+}) => {
+  const calculatedFontSize = fontSize ? fontSize : `calc(${size} * 0.5)`;
+  return (
+    <div
+      className={"mykn-iconinitials"}
+      style={{
```

Comment:
Can we 

- Use a a (container) tag P for (font) styling?
- Don't use inline styling, we can probably use the em unit for sizing? If we need a bigger size we can make a size prop.

Reply:
Good idea